free-2.md

```diff
@@ -58,12 +99,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Hopefully, the system wouldn't take too much time as the simulation can just be
+relegated to Pokémon Showdown. However, from experience I can guess working with
+that API will not be as simple as one would hope. It might be interesting to focus
+on the language. Since this is intended for comparisons, we might only specify what
+is different rather than everything like Smogon.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this project might be small in intention, but end up being quite
+large for the wrong reasons.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I don't think this would be a good project as it would probably end up being very
```

Comment:
I think there are a lot of interesting ways to engage with language design for this syntax, but you're right, in order to have a working language I think the implementation aspect might take up too much time.